exchanges/bybit/convert_types.go

```diff
@@ -0,0 +1,114 @@
+package bybit
+
+import (
+	"github.com/thrasher-corp/gocryptotrader/currency"
+	"github.com/thrasher-corp/gocryptotrader/encoding/json"
+	"github.com/thrasher-corp/gocryptotrader/types"
+)
+
+// Consts for WalletAccountType
+const (
+	Funding  WalletAccountType = "eb_convert_funding"
+	Uta      WalletAccountType = "eb_convert_uta"
+	Spot     WalletAccountType = "eb_convert_spot"
+	Contract WalletAccountType = "eb_convert_contract"
+	Inverse  WalletAccountType = "eb_convert_inverse"
+)
+
+// WalletAccountType represents the different types of wallet accounts
+type WalletAccountType string
+
+// CoinResponse represents a coin that can be converted
+type CoinResponse struct {
+	Coin               currency.Code `json:"coin"`
+	FullName           string        `json:"fullName"`
+	Icon               string        `json:"icon"`
+	IconNight          string        `json:"iconNight"`
+	AccuracyLength     int64         `json:"accuracyLength"`
+	CoinType           string        `json:"coinType"`
+	Balance            types.Number  `json:"balance"`
+	BalanceInUSDT      types.Number  `json:"uBalance"`
+	SingleFromMinLimit types.Number  `json:"singleFromMinLimit"` // The minimum amount of fromCoin per transaction
+	SingleFromMaxLimit types.Number  `json:"singleFromMaxLimit"` // The maximum amount of fromCoin per transaction
+	DisableFrom        bool          `json:"disableFrom"`        // true: the coin is disabled to be fromCoin, false: the coin is allowed to be fromCoin
+	DisableTo          bool          `json:"disableTo"`          // true: the coin is disabled to be toCoin, false: the coin is allowed to be toCoin
+	// Reserved fields, ignored for now
```

Comment:
Suggest we make them lowercase then so they don't get unmarshalled?

Reply:
I will drop the comment, if they get used later, then we don't need to open a PR to upper case them.